### DISCUSSION TITLE
How to set airflow.cfg in Docker Image

### DISCUSSION THREAD

**Discussion by DanielMorales9** [apache/airflow#19561] [Q&A]
Hi all, 

as per the discussion title, I could not find anywhere in the documentation how to set some configurations in the airflow.cfg. 
Is there an easy way to set them without overwrite the volume path?

**Comment by potiuk** [comment]
You can set all the configurations via environment variables- they will override what's in config (and you can dynamically set env vars when you start container): https://airflow.apache.org/docs/apache-airflow/stable/howto/set-config.html?highlight=configuration
